sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1487,12 +1585,16 @@ static void _event_cb(netdev_t *dev, netdev_event_t event)
     gnrc_netif_t *netif = (gnrc_netif_t *) dev->context;
 
     if (event == NETDEV_EVENT_ISR) {
+#if IS_USED(MODULE_GNRC_NETIF_EVENTS)
```

Comment:
Same here. What speaks againts:
```c
static inline gnrc_netif_events_post(gnrc_netif_t *netif)
{
#if IS_USED(GNRC_NETIF_EVENTS)
    event_post(&netif->evq, &netif->event_isr);
#else
    (void) netif;
#endif
}
```
?

Reply:
Nothing, implemented this in the latest fixup